vuepress/docs/v6.3.2/tutorials/samples/hub.md

```diff
@@ -86,17 +86,17 @@ Three roles are used to provide access to the Hub features.
 
 ### Bundle Group Version Status
 
-These are the possible statuses for the versions of a Bundle Group.
+These are the possible statuses for the versions of a Bundle Group:
 
-- Draft - this is the default status for the first version of a Bundle Group. 
-- Publication Request - an eh-author sets a version to this status to request an eh-manager or eh-admin to review the version and mark it for publication. An eh-manager or eh-admin may edit versions with this status.
-- Published - versions with this status are visible in the home page list of available Bundle Groups and also available in the App Builder-facing API. An eh-manager or eh-admin may edit Published versions.
-- Archived - previously Published versions are assigned this status. No edits can be made on an Archived version.
-- Deletion Request - an eh-manager or eh-admin can delete versions once this status has been set..
+- `Draft`: This is the default status for the first version of a Bundle Group. 
+- `Publication Request`: An eh-author sets a version to this status to request an eh-manager or eh-admin to review the version and mark it for publication. An eh-manager or eh-admin may edit versions with this status.
```

Comment:
'An eh-author sets a version to this status to request an eh-manager or eh-admin to review and mark it for publication.'   to reduce repeating 'version'